packages/tocco-ui/src/Icon/Icon.js

```diff
@@ -68,7 +74,11 @@ Icon.propTypes = {
   onClickFunction: PropTypes.func,
   onMouseEnterFunction: PropTypes.func,
   onMouseLeaveFunction: PropTypes.func,
-  onMouseDownFunction: PropTypes.func
+  onMouseDownFunction: PropTypes.func,
+  unicode: PropTypes.string
```

Comment:
Description is necessary here. And if i got that right, its only unicode OR icon that can be set?

Reply:
Yes, it is an exclusive or. Do you have an approach to test an XOR by propTypes. I have only found https://stackoverflow.com/questions/35892278/how-to-declare-react-proptypes-xor but it feels not right.

Reply:
Never used it before but i think custom proptype validators should do the job. But i would no use the generic method you mentioned and hard code instead